### ISSUE TITLE
Unable to run ignite-cli via npx or install via npm

### ISSUE THREAD

**Issue by brodie-pedalverse** [infinitered/ignite#2570]
### Describe the bug

**Environment**
- MacOS: Sonoma 14.1
- Node: v20.10.0
- Yarn: v1.22.21

**Describe the issue**
When I attempted to run the command: `npx ignite-cli@latest new PizzaApp`, it asks to install ignite-cli@9.0.2 but then terminate the process without any further steps
![IMTBN 2023-11-26 at 16 24 58](https://github.com/infinitered/ignite/assets/151793065/8fea2203-4229-477c-b5dc-ae4c5b8c41a9)

Then I attempted to install the package globally via npm, I got a bunch of errors
![IMTBN 2023-11-26 at 16 26 35](https://github.com/infinitered/ignite/assets/151793065/2b0f419b-d2df-407a-bc35-c96416028ffa)

```
npm install -g ignite-cli
npm ERR! code 1
npm ERR! path /Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/ignite-cli/node_modules/sharp
npm ERR! command failed
npm ERR! command sh -c (node install/libvips && node install/dll-copy && prebuild-install) || (node install/can-compile && node-gyp rebuild && node install/dll-copy)
npm ERR! sharp: Detected globally-installed libvips v8.15.0
npm ERR! sharp: Building from source via node-gyp
npm ERR!
npm ERR!   CC(target) Release/obj.target/nothing/../node-addon-api/nothing.o
npm ERR!   LIBTOOL-STATIC Release/nothing.a
npm ERR!   TOUCH Release/obj.target/libvips-cpp.stamp
npm ERR!   CXX(target) Release/obj.target/sharp-darwin-arm64v8/src/common.o
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@10.0.1
npm ERR! gyp info using node@20.10.0 | darwin | arm64
npm ERR! gyp info find Python using Python version 3.11.6 found at "/opt/homebrew/opt/python@3.11/bin/python3.11"
npm ERR! gyp info spawn /opt/homebrew/opt/python@3.11/bin/python3.11
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args '/Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args 'binding.gyp',
npm ERR! gyp info spawn args '-f',
npm ERR! gyp info spawn args 'make',
npm ERR! gyp info spawn args '-I',
npm ERR! gyp info spawn args '/Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/ignite-cli/node_modules/sharp/build/config.gypi',
npm ERR! gyp info spawn args '-I',
npm ERR! gyp info spawn args '/Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args '-I',
npm ERR! gyp info spawn args '/Users/brodienguyen/Library/Caches/node-gyp/20.10.0/include/node/common.gypi',
npm ERR! gyp info spawn args '-Dlibrary=shared_library',
npm ERR! gyp info spawn args '-Dvisibility=default',
npm ERR! gyp info spawn args '-Dnode_root_dir=/Users/brodienguyen/Library/Caches/node-gyp/20.10.0',
npm ERR! gyp info spawn args '-Dnode_gyp_dir=/Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args '-Dnode_lib_file=/Users/brodienguyen/Library/Caches/node-gyp/20.10.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args '-Dmodule_root_dir=/Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/ignite-cli/node_modules/sharp',
npm ERR! gyp info spawn args '-Dnode_engine=v8',
npm ERR! gyp info spawn args '--depth=.',
npm ERR! gyp info spawn args '--no-parallel',
npm ERR! gyp info spawn args '--generator-output',
npm ERR! gyp info spawn args 'build',
npm ERR! gyp info spawn args '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! Package libarchive was not found in the pkg-config search path.
npm ERR! Perhaps you should add the directory containing `libarchive.pc'
npm ERR! to the PKG_CONFIG_PATH environment variable
npm ERR! Package 'libarchive', required by 'vips', not found
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! warning: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/libtool: archive library: Release/nothing.a the table of contents is empty (no object file members in the library define global symbols)
npm ERR! ../src/common.cc:24:10: fatal error: 'vips/vips8' file not found
npm ERR! #include <vips/vips8>
npm ERR!          ^~~~~~~~~~~~
npm ERR! 1 error generated.
npm ERR! make: *** [Release/obj.target/sharp-darwin-arm64v8/src/common.o] Error 1
npm ERR! gyp ERR! build error
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack at ChildProcess.<anonymous> (/Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:209:23)
npm ERR! gyp ERR! System Darwin 23.1.0
npm ERR! gyp ERR! command "/Users/brodienguyen/.nvm/versions/node/v20.10.0/bin/node" "/Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd /Users/brodienguyen/.nvm/versions/node/v20.10.0/lib/node_modules/ignite-cli/node_modules/sharp
npm ERR! gyp ERR! node -v v20.10.0
npm ERR! gyp ERR! node-gyp -v v10.0.1
npm ERR! gyp ERR! not ok

npm ERR! A complete log of this run can be found in: /Users/brodienguyen/.npm/_logs/2023-11-26T05_26_40_212Z-debug-0.log
```

### Ignite version

9.0.2

### Additional info

Run: `npx ignite-cli doctor`

Output:
```bash
Need to install the following packages:
ignite-cli@9.0.2
Ok to proceed? (y)
```

**Comment by frankcalise** [comment]
Hey sorry for your troubles, can you try adding `--debug` to your new command to see if anything is happening at all?

**Comment by brodie-pedalverse** [comment]
> Hey sorry for your troubles, can you try adding `--debug` to your new command to see if anything is happening at all?

![IMTBN 2023-11-28 at 08 32 51](https://github.com/infinitered/ignite/assets/151793065/a45ee7e0-1be5-4a42-87d3-48c38e58d8e4)


Hi @frankcalise it doesn't seem to change anything at all

**Comment by frankcalise** [comment]
Have you tried another package manager by chance?

Can you normally run projects generated by the react-native or Expo CLI?

Has an older version of Ignite worked out for you, you could try npx targeting 8.11.3 for example?

**Comment by brodie-pedalverse** [comment]
Hi @frankcalise. Thanks for your time to get back to my issue.

Apparently, I have discovered some of the environment variables not being set properly from my machine. Thanks for this similar issue https://github.com/Homebrew/homebrew-core/issues/117642 everything seems to be working well after I set those env vars. Looking forward to testing out ignite-cli.

![IMTBN 2023-11-29 at 21 30 34](https://github.com/infinitered/ignite/assets/151793065/83078232-09b5-4d8f-8f44-b11e76f7fac5)

Appreciate your time to support me on this issue. Have a good one!
